### ISSUE TITLE
LBRY desktop app starts playing on its own

### ISSUE THREAD

**Issue by erkinalp** [lbryio/lbry-desktop#6023]
**Bug**
LBRY desktop app starts playing on its own when left in a paused video view.

**To Reproduce**
* Pick any video with autoplay settings on and pause it.
* Leave it for a few minutes.
* It starts playing on its own, even when minimised to tray.

**System Configuration**

- LBRY Daemon version: irrelevant as this is a UI bug
- LBRY App version: 0.50.2 AppImage (x86-64)
- LBRY Installation ID: irrelevant as not related to account balances
- Operating system: Ubuntu 20.04

**Additional context**

**Comment by tzarebczan** [comment]
Sorry to hear about the trouble. I'm not able to reproduce this. Do you see any errors on the screen when this happens? Is it usually within a short period amount of time, or after a while?

**Comment by erkinalp** [comment]
No, it just transistions from pause to play in 5-10 minutes. Even if it finished playing, in which case it starts over playing.

**Comment by tzarebczan** [comment]
Is it still happening? Can you keep an eye on what happens in the app when this does happen - i.e. are there any errors, refreshing? I think this is related to https://github.com/lbryio/lbry-desktop/issues/5376

**Comment by erkinalp** [comment]
Yes, can still reproduce.

**Comment by erkinalp** [comment]
And the problem is exactly the same. Network error, with or without error message, then continues playing.

**Comment by tzarebczan** [comment]
Can you take a screenshot of that when you see it?

**Comment by erkinalp** [comment]
also reproducible in 0.52.0 AppImage

**Comment by jomsai** [comment]
For the last month I have had this problem on Fedora 36 with LBRY_0.53.5.AppImage. It occurs several hours after I have left the app on pause and am usually asleep. That freaked me out the first time. The 2nd time I got out of bed and logged in to Fedora and closed the app and walked away. I then could hear 2 separate audio tracks going simultanously (one I paused and one I had already viewed). I confirmed the app was closed. I started HTOP and saw a lot of /tmp files with LBRY but did not see a lot of memory or CPU use by the LBRY app. I then shutdown for the night. There were no error messages. This has been an issue for a while as per messages here. Any new developments?
